src/util/Constants.js

```diff
@@ -111,6 +111,13 @@ function makeImageUrl(root, { format = 'webp', size } = {}) {
   if (size && !AllowedImageSizes.includes(size)) throw new RangeError('IMAGE_SIZE', size);
   return `${root}.${format}${size ? `?size=${size}` : ''}`;
 }
+/**
+ * Options for Image URLs.
+ * @typedef {Object} ImageURLOptions
+ * @property {string} [format]  One of `webp`, `png`, `jpg`, `gif`. If no format is provided,
```

Comment:
There's a double space between `[format]` and `One`.

Reply:
fixed in fa58ed0